libs/ui/src/components/molecules/DataIndicator/DataIndicator.styles.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import { ParMd } from '../../atoms';
+
+export const DataIndicatorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const IndicatorLabelMd = styled(ParMd)`
```

Comment:
Is this being used anywhere?

Reply:
Oht hat was supposed to replace my inline style that you commented on above.